src/detect-hostbits.c

```diff
@@ -78,7 +78,7 @@ void DetectHostbitsRegister (void)
 {
     sigmatch_table[DETECT_HOSTBITS].name = "hostbits";
     sigmatch_table[DETECT_HOSTBITS].desc = "operate on host flag";
-//    sigmatch_table[DETECT_HOSTBITS].url = DOC_URL DOC_VERSION "/rules/flow-keywords.html#flowbits";
+    sigmatch_table[DETECT_HOSTBITS].url = DOC_URL DOC_VERSION "/rules/xbits.html#unix-socket";
```

Comment:
this link is not very accurate. We're supposed to point to rule syntax and such, and this is more about unix-socket interaction